library/objective-c/EnvoyEngineImpl.m

```diff
@@ -284,6 +291,16 @@ - (int)registerFilterFactory:(EnvoyHTTPFilterFactory *)filterFactory {
   return kEnvoySuccess;
 }
 
+- (int)registerStringAccessor:(NSString *)name accessor:(EnvoyStringAccessor *)accessor {
+  // TODO(goaway): Everything here leaks, but it's all be tied to the life of the engine.
+  // This will need to be updated for https://github.com/lyft/envoy-mobile/issues/332
```

Comment:
Is this true in the context of a string accessor? Seems like it should be deallocated after usage since this is creating new strings each time it's accessed?

Reply:
yeah, still true. The accessor is the object being leaked, and it should be cleaned up.

The strings themselves get managed via the existing bridging utilities; the strings get boxed in envoy_data and that gets managed as an envoy buffer, which in turn cleans everything (all the way to the platform layer) when it is destroyed.

Reply:
got it, sounds good